spec/controllers/wishlists_for_guests_controller_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe WishlistsForGuestsController do
+  # let(:new_user) { User.create(email: "bob@example.org", password: "secret") }
+  # let(:new_wishlist) { Wishlist.create(name: "Name", description: "Description", user_id: new_user.id) }
+  let(:correct_token) { create(:wishlist).token }
+  let(:incorrect_token) { "000" }
```

Comment:
I think you don't need to use let, just put them directly, in incorrect token try maybe use `create(:wishlist, token: "000").token`